src/Configuration.cpp

```diff
@@ -64,9 +64,25 @@ void Configuration::begin()
 void Configuration::setModuleMode(ModuleMode f)
 {
   currentMode = f;
+  byte oldMode = storage->readEEPROM(0);
   storage->writeEEPROM(0, f);
 }
 
+void Configuration::setHeartbeat(bool beat)
```

Comment:
So this is implementing a "heartbeat on" flag rather than NOHEARTB as we have been discussed. The important part here is what the default is.

Reply:
Precisely!  As it stands at the moment, the default is no heartbeat which is at variance with the specified position.  It is easily changed if we decide to retain the specified default condition but an opportunity to review.